pkgs/build-support/docker/stream_layered_image.py

```diff
@@ -324,6 +324,7 @@ def main():
       else datetime.fromisoformat(conf["created"])
     )
     mtime = int(created.timestamp())
+    epoch = int(datetime.fromisoformat("1970-01-01T00:00:01Z").timestamp())
```

Comment:
So isn't this always `1` then, or is the integer mtime secretly in local time?

Also, nitpick, this isn't the UNIX epoch start, but 1 second after. I'm not getting anything useful searching for "Nix epoch", and I suggest we do not define that term.

```suggestion
    epochPlus1 = int(datetime.fromisoformat("1970-01-01T00:00:01Z").timestamp())
```